lib/apiservers/engine/backends/backends.go

```diff
@@ -299,7 +299,7 @@ func createImageStore(op trace.Operation) error {
 }
 
 // syncContainerCache runs once at startup to populate the container cache
-func syncContainerCache(op trace.Operation) error {
+func syncContainerCache(op trace.Operation, vs int64) error {
```

Comment:
I don't see `vs` being used anywhere - what is the argument for?

Reply:
Good catch. Removed it